scipy/stats/_hypotests.py

```diff
@@ -531,6 +531,12 @@ def cramervonmises(rvs, cdf, args=()):
     The null hypothesis is that the :math:`X_i` have cumulative distribution
     :math:`F`.
 
+    The statistic `T` is weighted by the sample size `n`, and is defined as:
```

Comment:
Single backticks (without explicit [role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#roles)) can be used for parameters, functions, and classes; double backticks are for (other, general) code. I think this is supposed to be math, so you need to specify the `:math:` role:
```suggestion
    The statistic :math:`T` is weighted by the sample size :math:`n`, and is defined as:
```

(Here and below)